pyvisa-py/tcpip.py

```diff
@@ -31,11 +31,6 @@ class TCPIPInstrSession(Session):
     using VXI-11
     """
 
-    lock_timeout = 1000
```

Comment:
I bet that support for those attributes should be added at some point. It may be worth replacing them by at least a comment saying that it should be done.

Reply:
Same attributes are set at after_parsing function. And used in form `self.<attr>`.
I didn't find any usage of these 4 values